cyences_app_for_splunk/default/macros.conf

```diff
@@ -23,7 +23,7 @@ definition = join notable_event_id type=left [| inputlookup cyences_notable_even
 | fillnull notable_event_status value="Unassigned"\
 | eval notable_event_assignee=if(isnotnull(notable_event_id) AND notable_event_id!="", notable_event_assignee, "BANANABANANA")\
 | eval notable_event_status=if(isnotnull(notable_event_id) AND notable_event_id!="", notable_event_status, "BANANABANANA")\
-| eval notable_event_selector=if(isnotnull(notable_event_id) AND notable_event_id!="", "-", "BANANABANANA")\
+| eval notable_event_selector=if(isnotnull(notable_event_id) AND notable_event_id!="", notable_event_id, "BANANABANANA")\
```

Comment:
Was this a bug?

Reply:
Yes, if you select the checkbox on first page then go to second page and then come to first page then selected checkboxes will deselected automatically but on edit it will make change for that id. 
so it's UI rendering issue.